### ISSUE TITLE
Property hsts is not allowed

### ISSUE THREAD

**Issue by epomatti** [Azure/azure-functions-host#6379]
I am trying to add HSTS header to my `hosts.json` but I get this message **"Property hsts is not allowed."**

![image](https://user-images.githubusercontent.com/6530769/87836900-1ef8d780-c868-11ea-9a58-2744fec73f3e.png)

```json
{
  "version": "2.0",
  "extensions": {
    "http": {
      "routePrefix": "api",      
      "hsts": {
        "isEnabled": true,
        "maxAge": 10,
        "includeSubDomains": true
      },
      "customHeaders": {
        "X-Content-Type-Options": "nosniff"
    }
    }
  },
  "extensionBundle": {
    "id": "Microsoft.Azure.Functions.ExtensionBundle",
    "version": "[1.*, 2.0.0)"
  }
}
```

But [HOST.JSON Bindings documentation](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-output#hostjson-settings) defines those properties.

![image](https://user-images.githubusercontent.com/6530769/87837203-1359e080-c869-11ea-90e0-6df849a59666.png)

I can confirm that the property is processed by the function host, as it gives an error if I specify a high "maxAge", however it is not working on the browser.

![image](https://user-images.githubusercontent.com/6530769/87837107-cbd35480-c868-11ea-8160-1c9e07c742fa.png)

My SDK is up to date:

```
$ func --version
3.0.2630
```

I'm also getting an error message for custom headers, but if add here it works. 

![image](https://user-images.githubusercontent.com/6530769/87837315-7a779500-c869-11ea-9e23-82bea4f94213.png)

**Comment by fabiocav** [comment]
@soninaren we see this commit here fixing the schema file: https://github.com/SchemaStore/schemastore/commit/9d4dc641f6763e9561c3793d1104399e40e801f5#diff-e47d5833465e07185fea991d46eea53f

Would you be able to check if this was correctly published? Looks like the tools are not picking that up. Does everything look correct to you here?

**Comment by soninaren** [comment]
@fabiocav Looks like the schema is incorrect. It needs to be under http-extension. https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/host.json#L122

**Comment by soninaren** [comment]
Addressed in PR: https://github.com/SchemaStore/schemastore/pull/1152

**Comment by soninaren** [comment]
Schema store changes are merged now. This should be fixed.

**Comment by epomatti** [comment]
@soninaren will it be available on the next core tools release? What are the steps that I can take to test it?

**Comment by fabiocav** [comment]
@epomatti once merged, those should go live pretty quickly, but for context, this issue shouldn't be blocking. Even if the validation is complaining, as long as you have the correct configuration, the feature should work as expected.

**Comment by epomatti** [comment]
@fabiocav I know it is nonblocking, for both hsts and the custom headers.

> I can confirm that the property is processed by the function host

I want to make sure it is solved, despite being non-critical it took me some time to find out the problem and finally open the issue.

It would also be nice to close the issues after the release, not prior to it, as it makes it easier for people to find it on the open issues, and it guarantees that whoever wants to test it, can test it.

**Comment by fabiocav** [comment]
@epomatti good to get confirmation. 

The typical flow in the Functions repos is to have issues closed once they're merged into their specific mainline branches. Release announcements and release notes include the list of issues in a given release.

For this particular issue, merging into the schema store site is the actual release, so the issue was indeed closed after that was done. You can see this is now live here: https://json.schemastore.org/host

**Comment by epomatti** [comment]
@fabiocav `hsts` was fixed, but `customHeaders` was not. I searched for "customHeders" in the schemastore but it doesn't have any matches.

> I'm also getting an error message for custom headers

Let's continue it at #6397.

Not ok for  `customHeaders`

![image](https://user-images.githubusercontent.com/6530769/88128604-b9308680-cbac-11ea-9ad8-e2ef69f1dc53.png)

OK for `hsts`

![image](https://user-images.githubusercontent.com/6530769/88128570-a6b64d00-cbac-11ea-9f59-b1910712f46a.png)
